src/common/example/index.html

```diff
@@ -38,25 +38,7 @@
 			}
 		}));
 
-		(function () {
-			var query = window.location.search.substring(1).split('&');
-			var module;
-
-			for (var i = 0; i < query.length; i++) {
-				var pair = query[i].split('=');
-				if (pair[0].toLowerCase() === 'module') {
-					module = pair[1].toLowerCase();
-					break;
-				}
-			}
-
-			require(['@dojo/shim/main'], function () {
-				require([ 'src/common/example/index' ], function () {});
-				if (module) {
-					require([ 'src/' + module + '/example/index' ], function () {});
-				}
-			});
-		})();
```

Comment:
This query string functionality is used by functional tests. We'll need to add it back in or figure out a different way to run them. I'm guessing CI fails because of this.

Reply:
Yeah @pottedmeat mentioned, this (seems strange that it would use the same index.html for both examples and tests imo) - we do need to think about the best way to deal with this, if we want to make the examples works as a real single projector example.